### DISCUSSION TITLE
Can we run finch as non root/admin

### DISCUSSION THREAD

**Discussion by rohitdadhich** [runfinch/finch#1168] [General]
Hello everyone,

I am running finch in my mac machine, when I run finch vm init command it asks me my admin credentials, I want to run finch as a non root user. Is it possible to do so ? If yes can I perform all finch operations as non root?

Thanks !

**Comment by henry118** [comment]
Hi @rohitdadhich 

Thanks for reaching out! 

You did raise a valid point that ideally users should be able to bootstrap finch without the root privilege. However this would require some effort in Finch's codebase to support the vzNAT networking in case of the vz VM type (which is the default setting today). 

I have created https://github.com/runfinch/finch/issues/1173 to track this feature request. We will try to get this change into future releases.

Thanks!

**Reply by rohitdadhich** [reply]
Thanks @henry118
